src/main/app/src/Forms/NewPurchaseOrderForm.js

```diff
@@ -94,6 +96,32 @@ export default function PurchaseOrderForm(props) {
   const [open, setOpen] = React.useState(false);
   const [vendor, setVendor] = React.useState(1);
 
+  const [openAlert, setOpenAlert] = React.useState(false);
+  const [alertText, setAlertText] = React.useState();
+
+  const AlertSnackbar = () => {
```

Comment:
This `AlertSnackbar` component is a duplicate of the component defined in `Planning.js` on line 55.

It would be cleaner to extract the logic for this alert into a separate shared component.